### ISSUE TITLE
Some download formats don't work to download as video

### ISSUE THREAD

**Issue by Reichmann-M** [fent/node-ytdl-core#910]
I'm writing a simple youtube downloading tool on an express server and most of the videos I'm testing with are fine.

My script turns out the best video format with ytdl.getInfo and when i'm trying to download this with ytdl.downloadFromInfo and save it to file with fs.createWriteStream i'm getting a 0 bytes file. Other formats work for this video and download is fine.

Example video I used: owic2j9FcMg
itags 303 and 299 don't work, which both have the best quality.

Even with the quality option: 'highestvideo' it returns a 0 byte video.

I hope you can help me with that :)

**Comment by redbrain** [comment]
I couldn't reproduce the issue with itag 303, but I could with itag 299, which appears to silently 404.
This may be a bug with the library.

**Comment by Reichmann-M** [comment]
> I couldn't reproduce the issue with itag 303, but I could with itag 299, which appears to silently 404.
> This may be a bug with the library, but could also be YouTube's mistake.

yeah same happened to me when i tried to open the urls returned in formats

**Comment by KirianCaumes** [comment]
Same problem here, any updates? :/
